src/pages/NotePage/Notes.jsx

```diff
@@ -1,13 +1,86 @@
 import "./Notes.css"
 import { NavBar, Aside } from "../../components"
 import { useState } from "react";
-
+import { useData } from "../../context/data-context";
+import { useAuth } from "../../context/auth-context";
+import { postNote,editNote,deleteNote } from "../../services/Services";
+import { NoteCard } from "../../components";
 
 export const Notes=()=>{
 const [noteCard,setNoteCard] = useState(false);
+
 const expandCard=()=>{
     setNoteCard(true);
 }
+const date = new Date();
+const initialVal= {
+    _id:"",
+    title:"",
+    body:"",
+    backGround:"#e7dcdc",
+    CreatedAt:`${date.getDate()}/0${
```

Comment:
Adding a 0 before every month can cause the double digit months to displayed as 12/010/2022. You could change it to some other approach. refer this https://bobbyhadz.com/blog/javascript-get-month-and-date-in-2-digit-format